src/main/java/core/basesyntax/service/strategy/ActivityService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.utility.FruitTransaction;
+
+public interface ActivityService {
+    int execute(FruitTransaction transaction);
```

Comment:
I would not call that a service it is the template to implement logic of handling some operation, so 
```suggestion
public interface OperationHandler {
    int handle(FruitTransaction transaction);
```
Word `service` is usualy used when you have some logic in relation to some entity, like `UserService`, `FruitService`, `BookService`, `ProductService` etc. there are other cases as well but mostly it is used like that,
There are plenty other words that  we can use to define waht the class does, see the next comments